app/Http/Resources/User.php

```diff
@@ -94,6 +95,11 @@ public function toArray($request): array
                     $this->resource->relationLoaded('roles') ? $this->getAllPermissions()->pluck('name') : [],
                 ]
             ),
+            'travel' => $this->when(
+                ($this->requestingSelf($request) || Auth::user()->can('manage-travel')) &&
```

Comment:
It seems like authorize include wants `read-users` but this if statement is looking for `manage-travel`, am I missing something?

Reply:
I guess the end result is you need both permissions. `read-users` is required to load the permission in the controller, `manage-travel` is required for it to populate in the resource here.

It's a little convoluted but I don't super want to add a `read-travel` permission just for this.

Reply:
this seems like jank that might end up confusing someone later on. I would lightly prefer (4/10) if we did something to make it more clear what's happening. AuthorizeIncludes is a bit hard to understand for newer devs or if you haven't touched the code in a while and forgot it exists

Reply:
Fair enough. Let me play around with it a bit.